### PR TITLE
Tests matcher spec

### PR DESCRIPTION
**Pull request by besSveta**
tests for amr matcher

### CODE REVIEW

File: metta_amr/metta_space/metta_space.py [hunk 1]
```diff
@@ -180,6 +180,10 @@ def match_concept(input, template):
             else:
                 return input[:meaning_pos.start(0)] == template
 
+    def get_concepts_of_instance(self, amr_instance):
```
Comment (besSveta) [inline metta_amr/metta_space/metta_space.py:183]: this method was used in test_spec_amrmatcher.py

File: metta_amr/metta_space/metta_space.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def get_amrsets_by_concept(self, concept):
     @staticmethod
     def is_a(value, type):
         if type == Types.AmrVariable:
-            return value.startswith("(Var")
+            return value.startswith("(Var") or value == "*"
```
Comment (besSveta) [inline metta_amr/metta_space/metta_space.py:81]: found this omission thanks to tests

File: metta_amr/experiments/unit_tests.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 import pathlib
 
 T = unittest.TestCase
-from metta_space import PatternLoader, MettaSpace
+from metta_space import PatternParser, MettaSpace
```
Comment (besSveta) [inline metta_amr/experiments/unit_tests.py:7]: renamed according to awakening dialogue system

File: metta_amr/metta_space/pattern_parser.py [hunk 4]
```diff
@@ -42,6 +43,13 @@ def _process_triples(self, triples):
             # else:
             self.amr_space.add_triple(triple)
 
+    def load_templates_from_files(self, filenames):
+        if not isinstance(filenames, list):
+            filenames = [filenames]
+        for filename in filenames:
+            self.load_templates_from_file(filename)
+        self.amr_space.index_amrsets()
```
Comment (besSveta) [inline metta_amr/metta_space/pattern_parser.py:51]: I've add this function because we need to call index_amrsets() after all  templates were loaded

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**